### ISSUE TITLE
cmalign phase omitting some seqs from output

### ISSUE THREAD

**Issue by dhoogest** [nhoffman/dada2-nf#25]
Noting this as an observation for now:

The attached `seqs.fasta` was produced from a set of clinical reads and contains 270 unique svs (see #22 for discussion about the SV labelling). Not all reads are preserved in the output `sv_aln_scores.txt` however, which looks like an expectation of the downstream `filter_16s.py` script. Among the sequences in this file are synthetic control sequences, which might fail certain assumption of the alignment model (speculation). For instance sv-00001:

[seqs.fasta.zip](https://github.com/nhoffman/dada2-nf/files/5876600/seqs.fasta.zip)

```
>sv-000001:1029-38
CAGGCCTAACACATGCAAGTCGAACGGTTTGCCACAGATACGTACCGCTCATAACGCGAACCGAAGCGCAGTAGAAGTACTCCGTATCCTACCTCGGTCGTGGTTTAGGCTATCGACATCTTGCATGGGCTTCCCTAGTGAACTCTTGGGATGTATGGTACACTGGTGTAAGACATCAGACAACGAACGCTAACTATCCTGCAACACTCCGGAAGTCTACTAGTCGCCGACGCGATACGAAACTTCGGAGTGCTATTGTGCGTACCAGATCCAGTATCACT
```

**Comment by dhoogest** [comment]
@crosenth figured you're the best poised to look into this one. @mwohl cc

**Comment by dhoogest** [comment]
I guess I should rewrite the title, since refactoring `filter_16s.py ` to incorporate sequences with no hits (assuming that's the root cause) into the logic would presumably also work

**Comment by dhoogest** [comment]
Note that the bug fix in `filter_16s` in #26 allows for pipeline to complete. However, synthetic 16S sequences (incorporated in the NGS16S assay as internal standards) were still failing to align to model.  These same SVs do not fail alignment to the current model used in the `NGS16S` pipeline. To accomodate them, I added the alternate model to `/data` and extended the params files to specify the alignment profile. See also https://gitlab.labmed.uw.edu/molmicro/NGS16S/-/issues/186
